src/Integration.UnitTests/ProfileConflicts/RuleSetConflictsControllerTests.cs

```diff
@@ -0,0 +1,245 @@
+            this.host.VisualStateManager.IsBusy = false;
+            this.host.VisualStateManager.SetBoundProject(new Integration.Service.ProjectInformation());
+            var section = ConfigurableSectionController.CreateDefault();
+            this.host.SetActiveSection(section);
+            ConfigurableUserNotification notifications = (ConfigurableUserNotification)section.UserNotifications;
+
+            ProjectRuleSetConflict[] conflicts = new[] { ConfigurableConflictsManager.CreateConflict() };
+
+            RuleSet fixedRuleSet = TestRuleSetHelper.CreateTestRuleSet(3);
+            fixedRuleSet.FilePath = "MyFixedRules.ruleset";
+
+            RuleSetInspectorTestDataProvider inspectorData = new RuleSetInspectorTestDataProvider();
+            var weakenedRulesMap = new Dictionary<RuleReference, RuleAction>();
+            inspectorData.FindConflictsResult = new RuleConflictInfo(new RuleReference[0], weakenedRulesMap);
+            inspectorData.FixConflictsResult = new FixedRuleSetInfo(fixedRuleSet, new[] { "reset.ruleset" }, new[] { "deletedRuleId1" });
+            this.ruleSetInspector.FindConflictingRulesAction = inspectorData.FindConflictingRulesAction;
+            this.ruleSetInspector.FixConflictingRulesAction = inspectorData.FixConflictingRulesAction;
+
+            ICommand fixMeCommand = testSubject.CreateFixConflictsCommand(conflicts);
+            section.UserNotifications.ShowNotificationWarning("fix me", NotificationIds.RuleSetConflictsId, fixMeCommand);
+
+            // Act
+            fixMeCommand.Execute(null);
+
+            // Verify
+            this.sccFS.AssertFileExists(fixedRuleSet.FilePath);
+            this.rsSerializer.AssertRuleSetsAreSame(fixedRuleSet.FilePath, fixedRuleSet);
+            this.outputWindow.AssertOutputStrings(1);
+            this.outputWindow.AssertMessageContainsAllWordsCaseSensitive(0, 
+                words: new[] { fixedRuleSet.FilePath, "deletedRuleId1", "reset.ruleset" },
+                splitter:new[] {'\n', '\r', '\t', '\'', ':' });
+            notifications.AssertNoNotification(NotificationIds.RuleSetConflictsId);
+        }
+        #endregion
+
+        #region Helpers
+        private void ConfigureServiceProviderForFixConfclitsCommandExecution()
+        {
+            this.ruleSetInspector = new ConfigurableRuleSetInspector();
+            this.serviceProvider.RegisterService(typeof(IRuleSetInspector), this.ruleSetInspector);
+
+            this.sccFS = new ConfigurableSourceControlledFileSystem();
+            this.serviceProvider.RegisterService(typeof(ISourceControlledFileSystem), this.sccFS);
+
+            this.rsSerializer = new ConfigurableRuleSetSerializer(sccFS);
+            this.serviceProvider.RegisterService(typeof(IRuleSetSerializer), this.rsSerializer);
+        }
+
+        private class RuleSetInspectorTestDataProvider
+        {
+            public RuleConflictInfo FindConflictingRulesAction(string baseline, string project, string[] directories)
```

Comment:
A region/comments saying which methods on this class are called from product code and which are test helpers would help.